recipe/meta.yaml

```
@@ -35,10 +35,11 @@ requirements:
     # proj and geos are needed as dependencies even though not listed upstream
     - proj
     - geos
+    # including matpltolib-base even though it's optional upstream
```

Typo "matpltolib"

---

I'm pretty sure that is correct. Isn't that the library used to plot ptolemy's model of the universe?

---

```suggestion
    # including matplotlib-base even though it's optional upstream
```
Aargh!  I can't type today.  Thanks @dopplershift